README.md

```diff
@@ -1,18 +1,34 @@
-# Node sample
+# Node Sample App
 
-A sample node microservice block for [kintohub](http://kintohub.com).
+A sample node microservice block for [Kintohub](http://kintohub.com)
 
+## :rocket: Deploying on Kintohub v1
 
-# First time setup
+### # Using Dockerfile 
 
-* run `npm install`
+- **Type:** Web App
+- **Repo:** https://github.com/kintohub/node-example
+- **Branch:** master
+- **Language:** Dockerfile
+- **Dockerfile name:** Dockerfile
+- **Subfolder Path:** `.`
+- **Port:** 80
 
 
-# Run
+<hr>
 
-* `npm start` run on port 8000
-* `npm run prod` run on port 80
+### # Traditional way
 
-# Support
+- **Type:** Web App
+- **Repo:** https://github.com/kintohub/node-example
+- **Branch:** master
+- **Language:** NodeJS
+- **Language version:** 14
+- **Build Command:** `npm install`
+- **Start Command:** `npm run prod`
+- **Subfolder Path:** `.`
+- **Port:** 80
+
```

Comment:
action - add running locally with the commands to start it locally and what you require to install (node, etc)